habitat_sim/utils/datasets_download.py

```diff
@@ -103,8 +103,8 @@ def prompt_yes_no(message):
 
 def clean_data(uid):
     r"""Deletes the "root" directory for the named data-source."""
-    if uid not in data_sources:
-        print("Data download failed, no datasource named " + uid)
+    if not data_sources.get(uid):
```

Comment:
Wait, this works? The old method was much more readable, see the other comment for what I meant.